jib-gradle-plugin/build.gradle

```diff
@@ -98,6 +98,13 @@ tasks.withType(JavaCompile) {
     }
 }
 
+// Fail build on javadoc warnings
+if (JavaVersion.current().isJava8Compatible()) {
```

Comment:
We can just configure this directly on `googleJavaFormat` right?

Reply:
We should also add this for core and maven.

Reply:
How (for both of those)?

Reply:
It might not be possible but it looks like the gradle plugin can define options as part of the extension: https://github.com/sherter/google-java-format-gradle-plugin#extended-usage

Reply:
Oh okay that's probably my misunderstanding